site/documentation/content/api/device-registration/index.md

```diff
@@ -60,7 +60,7 @@ In case of a successful invocation of the operation, the body of the response me
 | Name             | Mandatory | JSON Type     | Description |
 | :--------------- | :-------: | :------------ | :---------- |
 | *device-id*      | *yes*     | *string*      | The ID of the device that is subject of the assertion. |
-| *via*            | *no*      | *array*       | The IDs (JSON strings) of gateways which may act on behalf of the device. This property MUST be set if any gateways are registered for the device. If the assertion request contained a *gateway_id* property and the response's *status* property has value `200` (indicating a successful assertion) then the array MUST at least contain the gateway ID from the request. |
+| *via*            | *no*      | *array*       | The IDs (JSON strings) of gateways which may act on behalf of the device. This property MUST be set if any gateways are registered for the device. This property does not include gateway groups that may be registered to act on behalf of the device. Instead, this property may contain devices that are member of one of the groups that is registered to act for the device. If the assertion request contained a *gateway_id* property and the response's *status* property has value `200` (indicating a successful assertion) then the array MUST at least contain the gateway ID from the request.
```

Comment:
FMPOV this change is not necessary as the protocol adapters should not need to bother with gateway groups or any such thing. The whole concept of GW groups should be encapsulated in the device registry implementation FMPOV.
The adapters have been expecting to receive GW IDs and that is what they continue to do. If possible, I would like to confine the definition/mentioning of GW groups to the Device Registry Management API which is the place where I think we should start this discussion as it needs to be adapted to support the *memberOf* property etc, isn't it?